src/b-splines/cubic.jl

```diff
@@ -173,75 +258,62 @@ function prefiltering_system{T,TC}(::Type{T}, ::Type{TC}, n::Int,
 end
 
 """
-`Line` `OnGrid` amounts to setting `y_0''(x) = 0` at `x = 0`. Applying this
+`Cubic{Line}` `OnGrid` amounts to setting `y_1''(x) = 0` at `x = 1`. Applying this
 condition gives:
 
-    1 cm -2 c + 1 cp = 0 = 0
-
-This is the same system as `Quadratic{Line}`, `OnGrid` so we reuse the
-implementation
+    1 cm -2 c + 1 cp = 0
 """
-function prefiltering_system{T,TC,GT<:GridType}(::Type{T}, ::Type{TC}, n::Int,
-                                                ::Type{Cubic{Line}}, ::Type{GT})
-    prefiltering_system(T, TC, n, Quadratic{Line}, OnGrid)
+function prefiltering_system{T,TC}(::Type{T}, ::Type{TC}, n::Int,
```

Comment:
Oh man ok. Good catch. Sounds good, it's nice to have it below he description also